Gopkg.toml

```diff
@@ -43,16 +47,16 @@
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
   name = "github.com/giantswarm/operatorkit"
 
 [[constraint]]
   name = "github.com/google/go-cmp"
   version = "0.3.1"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/prometheus/client_golang"
+  version = "0.9.3"
```

Comment:
This is aligned with other projects. e.g. https://github.com/giantswarm/cert-operator/blob/f839898abcb290770d87ab425f4a55930c6bcb4a/Gopkg.toml#L74